### PR TITLE
Update ROADMAP.md

### PR DESCRIPTION
**Pull request by IlCallo**
**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [ ] Feature
- [x] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs)  or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -45,15 +45,16 @@ We are updating this section constantly in order to keep you up to date with our
...
-* 🚧 (WIP) Quasar website revamp 🔥
+* ✌️ (DONE) Triaging enhancements ([reference](https://github.com/quasarframework/rfcs/issues/10))
 * ✌️ (DONE) Support Cypress Component Testing into Quasar Cypress AE ([reference](https://github.com/quasarframework/quasar-testing/issues/163) [reference](https://github.com/quasarframework/quasar-testing/pull/185) [reference](https://github.com/quasarframework/quasar/discussions/11496))
-* 🚧 (WIP) Enhance Quasar types to be more precise, provide guidelines for community contributions ([reference](https://github.com/quasarframework/quasar/issues/8642) [reference](https://github.com/quasarframework/quasar/issues/8493) [reference](https://github.com/quasarframework/quasar/issues/11090) [reference](https://github.com/quasarframework/quasar/issues/11043))
+* ✌️ (DONE) Enhance Quasar types to be more precise, provide guidelines for community contributions ([reference](https://github.com/quasarframework/quasar/issues/8642) [reference](https://github.com/quasarframework/quasar/issues/8493) [reference](https://github.com/quasarframework/quasar/issues/11090) [reference](https://github.com/quasarframework/quasar/issues/11043))
```
Comment (yusufkandemir) [inline ROADMAP.md]:
```suggestion
* ✌️ (DONE) Enhance Quasar types to be more precise, provide guidelines for community contributions ([#8642](https://github.com/quasarframework/quasar/issues/8642), [#8493](https://github.com/quasarframework/quasar/issues/8493), [#11090](https://github.com/quasarframework/quasar/issues/11090), [#11043](https://github.com/quasarframework/quasar/issues/11043))
```

**Before:**
image

**After:**
image

File: ROADMAP.md [hunk 2]
```diff
@@ -45,15 +45,16 @@ We are updating this section constantly in order to keep you up to date with our
...
 
 ### Q1 2022
+* ✌️ (DONE) Fix intellisense problem for packages provided by `@quasar/app` ([reference](https://github.com/quasarframework/quasar/issues/9235))
```
Comment (yusufkandemir) [inline ROADMAP.md]:
```suggestion
* ✌️ (DONE) Fix intellisense problem for packages provided by `@quasar/app` ([#9235](https://github.com/quasarframework/quasar/issues/9235))
```

File: ROADMAP.md [hunk 3]
```diff
@@ -45,15 +45,16 @@ We are updating this section constantly in order to keep you up to date with our
...
+* 🚧 (WIP) Quasar website revamp 🔥
 * 🚧 (WIP) Video tutorials about all Quasar components (~~pre-access will be a paid service, will be later released for free~~ will be released for free, but register to https://quasarcast.com/course for future paid advanced Quasar courses 🙂) ([reference](https://quasarcomponents.com/))
 * 🚧 (WIP) Add automatic tests to Quasar UI ([reference](https://github.com/Evertvdw/quasar/tree/unit-testing))
-* Fix intellisense problem for packages provided by `@quasar/app` ([reference](https://github.com/quasarframework/quasar/issues/9235))
+* More triaging automation ([reference](https://github.com/quasarframework/rfcs/issues/10))
```
Comment (yusufkandemir) [inline ROADMAP.md]:
```suggestion
* More triaging automation ([quasarframework/rfcs#10](https://github.com/quasarframework/rfcs/issues/10))
```

File: ROADMAP.md [hunk 4]
```diff
@@ -45,15 +45,16 @@ We are updating this section constantly in order to keep you up to date with our
 ### Q4 2021
 * ✌️ (DONE) Volar support ([reference](https://github.com/quasarframework/quasar/discussions/10619))
 * ✌️ (DONE) Vite.js plugin for Quasar v2 ([reference](https://github.com/quasarframework/quasar/issues/7815))
-* 🚧 (WIP) Triaging enhancements and automation ([reference](https://github.com/quasarframework/rfcs/issues/10))
-* 🚧 (WIP) Quasar website revamp 🔥
+* ✌️ (DONE) Triaging enhancements ([reference](https://github.com/quasarframework/rfcs/issues/10))
```
Comment (yusufkandemir) [inline ROADMAP.md]:
```suggestion
* ✌️ (DONE) Triaging enhancements ([quasarframework/rfcs#10](https://github.com/quasarframework/rfcs/issues/10))
```

**Review by yusufkandemir — APPROVED** [review]
LGTM 👍 I added some suggestions about reference links for optional improvement.

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**